native/cocos/platform/SDLHelper.cpp

```diff
@@ -160,43 +160,43 @@ int SDLHelper::init() {
     return 0;
 }
 
-void SDLHelper::dispatchWindowEvent(IEventDispatch *delegate, uint32_t windowId, const SDL_WindowEvent &wevent) {
+void SDLHelper::dispatchWindowEvent(uint32_t windowId, const SDL_WindowEvent &wevent) {
     WindowEvent ev;
     ev.windowId = windowId;
 
     switch (wevent.event) {
         case SDL_WINDOWEVENT_SHOWN: {
             ev.type = WindowEvent::Type::SHOW;
-            delegate->dispatchEvent(ev);
+            cc::event::broadcast<cc::events::WindowEvent>(ev);
```

Comment:
It is not recommended to send messages directly here. The purpose of this design is to encapsulate the event distribution in a higher level. If you change it, you can just change the base class directly.
According to the current modification, it is necessary to change all platforms, which is not conducive to cross-platform